openpype/hosts/traypublisher/plugins/publish/validate_frame_ranges.py

```diff
@@ -0,0 +1,72 @@
+import re
+
+import pyblish.api
+
+import openpype.api
+from openpype import lib
+from openpype.pipeline import (
+    PublishXmlValidationError,
+    OptionalPyblishPluginMixin
+)
+
+
+class ValidateFrameRange(OptionalPyblishPluginMixin,
+                         pyblish.api.InstancePlugin):
+    """Validating frame range of rendered files against state in DB."""
+
+    label = "Validate Frame Range"
+    hosts = ["traypublisher"]
+    families = ["render"]
+    order = openpype.api.ValidateContentsOrder
+
+    optional = True
+    # published data might be sequence (.mov, .mp4) in that counting files
+    # doesnt make sense
+    check_extensions = ["exr", "dpx", "jpg", "jpeg", "png", "tiff", "tga",
+                        "gif", "svg"]
+    skip_timelines_check = []  # skip for specific task names (regex)
+
+    def process(self, instance):
+        # Skip the instance if is not active by data on the instance
+        if not self.is_active(instance.data):
+            return
+
+        if any(re.search(pattern, instance.data["task"])
```

Comment:
This logs out that it's skipping but actually is not.

Reply:
Added condition to check only if `skip_timelines_check` is configured.